pilot/pkg/model/service.go

```diff
@@ -1030,7 +1030,7 @@ const (
 	WaypointAccepted ConditionType = "WaypointAccepted"
 )
 
-type ConditionSet = map[ConditionType][]Condition
+type ConditionSet = map[ConditionType]*Condition
```

Comment:
Technically not strictly needed, but the reason we made it a list is no longer applicable and it makes it harder to make the change if its a list